blogposts/2021/release-3.25.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Sourcegraph 3.25 release"
+publishDate: 2021-02-20T10:00-07:00
+description: "Sourcegraph 3.25 is released."
+tags: [blog, release]
+slug: "release/3.25"
+published: true
+changelogItems:
+- description: "Go `1.15` introduced changes to SSL/TLS connection validation which requires certificates to include a `SAN`. This field was not included in older certificates and clients relied on the `CN` field. You might see an error like `x509: certificate relies on legacy Common Name field`. We recommend that customers using Sourcegraph with an external database and and connecting to it using SSL/TLS check whether the certificate is up to date. AWS RDS customers please reference [AWS' documentation on updating the SSL/TLS certificate](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL-certificate-rotation.html) for steps to rotate your certificate."
+  category: Admin
+  url: https://github.com/sourcegraph/sourcegraph/pull/18024
+# Added
+- description: "New site config option `\"log\": { \"sentry\": { \"backendDSN\": \"\u003cREDACTED\u003e\" } }` to use a separate Sentry project for backend errors."
+  url: https://github.com/sourcegraph/sourcegraph/pull/17363
+  category:
+- description: Structural search now supports searching indexed branches other than default.
+  url: https://github.com/sourcegraph/sourcegraph/pull/17726
+  category: Search
+- description: Structural search now supports searching unindexed revisions.
+  url: https://github.com/sourcegraph/sourcegraph/pull/17967
+  category: Search
+- description: "New site config option `\"allowSignup\"` for SAML authentication to determine if automatically create new users is allowed."
+  url: https://github.com/sourcegraph/sourcegraph/pull/17989
+  category:
+- description: "Experimental: The webapp can now stream search results to the client, improving search performance. To enable it, add `{ \"experimentalFeatures\": { \"searchStreaming\": true } }` in user settings."
```

Comment:
@christinaforney @content-hacker - how do you feel about including experimental features in the changelog?

Reply:
It might be helpful to use different terminology. The changelog is what lives in our sourcegraph repo (what this was generated from) and should have everything added to the product. The blog version of the changelog is the distilled version of this with the 5-7 highlights for a release.

I think it can be fun to highlight experimental features - we can use it to get more users to turn on and test things for us, and shows incremental progress. We don't need to show every experimental thing though and should consider if this outcome is what we want for each thing.

Reply:
@christinaforney - that was my mistake. I was referring to both the changelog and the release post as the "changelog." I was referring to including this in the release post.

Reply:
I thought so :) my second paragraph hopefully answers that question for you?

Reply:
Yes - thank you!

Reply:
I like this.

@content-hacker - any recommendation for indicating that it's experimental?

Reply:
I think we should keep it. Is experimental the right word? Is it too small of a change to use `beta` or is it too jarring to use a CTA (Try now: XYZ)

Just thoughts.. I have no problem with keeping experimental.